packages/core/src/symbol/SymbolUtil.ts

```diff
@@ -972,7 +972,7 @@ type QueryMemberCallback = (this: void, query: SymbolQuery) => unknown
 /* istanbul ignore next */
 export class SymbolQuery {
 	readonly category: string
-	readonly path: readonly string[]
+	path: readonly string[]
 	readonly #doc: TextDocument
```

Comment:
what does a hash `#` prefix indicate for class properties?

Reply:
[private property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties)